index.html

```diff
@@ -1734,28 +1781,77 @@ <h3>Document Encoding</h3>
+          is not one of the namespaces listed in <a href="#namespaces"></a> and
+          the set of attributes in the global namespace that are not otherwise defined in DAPT or in [[TTML2]].</p>
+        <div class="note">
+          <p>The requirements in this section are intended to facilitate forwards and backwards compatibility,
+            specifically to permit:
+          </p>
+          <ul>
+            <li>DAPT processors targeted at one version of the specification to process <a>DAPT documents</a> that
+              include vocabulary or semantics defined in future versions,
+              albeit without supporting the later features;</li>
+            <li>DAPT processors targeted at one version of the specification to process <a>DAPT documents</a>
+              authored for an earlier version with similar or identical behaviour to a DAPT processor
+              targeted at that earlier version.</li>
+          </ul>
+        </div>
+        <p class="note">
           <a>DAPT Documents</a> remain subject to the content conformance requirements specified at Section 3.1 of [[TTML2]].
-          In particular, a <a>DAPT Document</a> can contain elements and attributes not in any TT namespace, i.e. in foreign namespaces, since such elements and attributes are pruned by the algorithm at Section 4 of [[TTML2]] prior to evaluating content conformance.
+          In particular, a <a>DAPT Document</a> can contain <a>foreign vocabulary</a>,
+          since such elements and attributes are pruned by the algorithm at Section 4 of [[TTML2]] prior to evaluating content conformance.
         </p>
         <p class ="note">
-          For validation purposes it is good practice to define and use a content specification for all foreign namespace elements and attributes used within a <a>DAPT Document</a>.
+          For validation purposes it is good practice to define and use
+          a content specification for all <a>foreign vocabulary</a> used within a <a>DAPT Document</a>.
         </p>
         <p>
-          A <a>transformation processor</a> SHOULD preserve such elements or attributes whenever possible.
+          A <a>transformation processor</a> SHOULD preserve <a>foreign vocabulary</a> associated with or descendant to
+          <code>&lt;metadata&gt;</code> elements even if it is not associated with features that it supports.</p>
+        <p>A <a>transformation processor</a> MUST prune vocabulary that is a) not associated or descendant to
+          <code>&lt;metadata&gt;</code> elements and b) not associated with any feature that it supports.</p>
+        <p class="note">See also <a href="#ttp-contentprofiles"></a> which prohibits the signalling of profile
+          conformance to profiles that the <a>transformation processor</a> does not support.</p>
+        <p>
+          After attribute value computation,
+          a <a>presentation processor</a> SHOULD ignore elements and attributes that it does not recognise and support.
```

Comment:
Happy to add the link to §6.4.

Re switching the sense, the risk is that's too inclusive a set; I don't want to tell implementers that they must compute attribute values on all elements even if that computation is not actually needed, e.g. because the attribute is specified on an element that it applies to, so there is no need to compute the value on a parent element to which the attribute does not apply.

Reply:
Link added in f8c86e4.